package.json

```diff
@@ -1,29 +1,32 @@
 {
-  "name": "cli",
-  "version": "1.0.0",
-  "description": "Automate your GitHub workflows using Azure CLI scripts.",
-  "main": "lib/main.js",
-  "scripts": {
-    "build": "tsc",
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/Azure/CLI.git"
-  },
-  "author": "Microsoft",
-  "license": "MIT",
-  "devDependencies": {
-    "@types/node": "^12.7.11",
-    "typescript": "^3.6.3"
-  },
-  "dependencies": {
-    "@actions/core": "^1.1.3",
-    "@actions/exec": "^1.0.1",
-    "@actions/io": "^1.0.1"
-  },
-  "bugs": {
-    "url": "https://github.com/Azure/CLI/issues"
-  },
-  "homepage": "https://github.com/Azure/CLI#readme"
-}
+    "name": "cli",
+    "version": "1.0.0",
+    "description": "Automate your GitHub workflows using Azure CLI scripts.",
+    "main": "lib/main.js",
+    "scripts": {
+        "build": "tsc",
+        "test": "echo \"Error: no test specified\" && exit 1"
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/Azure/CLI.git"
+    },
+    "author": "Microsoft",
+    "license": "MIT",
+    "devDependencies": {
+        "@types/node": "^12.7.11",
+        "typescript": "^3.6.3"
+    },
+    "optionalDependencies": {
+        "credscan-pkg": "file:credscan-pkg-1.0.0.tgz"
```

Comment:
are we using tgz file, or install from code?

Reply:
install from code?

Reply:
How this file:credscan-pkg-1.0.0.tgz gets generated?

Reply:
using `npm pack`

Reply:
Where this command is getting run